be/src/storage/tablet_updates.cpp

```diff
@@ -1332,6 +1350,8 @@ void TabletUpdates::_erase_expired_versions(int64_t expire_time,
     for (int i = 0; i < _apply_version_idx; i++) {
         if (_edit_version_infos[i]->creation_time <= expire_time) {
             expire_list->emplace_back(std::move(_edit_version_infos[i]));
+        } else {
+            break;
```

Comment:
need this print a log just like the two other changes since that they are the same scene.

Reply:
no it's not